### PR TITLE
[Documentation] Fix ambiguities of TypeScript and C# documentation for manifests

### LINKED ISSUE #3638: [Docs] Ambiguities in the docs (Project does not have a "wwwroot" folder; How to publish?)
/skills/tutorials/create-skill/typescript/5-update-manifest/

I followed the steps from the TS create skill tutorial and in step 5 there are some ambiguities.

* I can not find a folder wwwroot in my project. Of course, there is a "src/mainfest/" folder with a "mainfest-1.0.json" sample file in it. But when I deploy or publish I can not access the manifest from the browser. There is just a 404 JSON response `{"code":"ResourceNotFound","message":"/manifest/manifest-1.1.json does not exist"}`
* Step 2: It is hard to know what exactly is `{YOUR_SKILL_URL}`, where did I defined it? Previous steps only talk about "name" or "id". Can I find it in the "appsettings.json"? Did the deployment script create it for me?
* Step 3: How to publish? There are no instructions how I should publish my changes. Guess you should not call the Deploy-Script from https://microsoft.github.io/botframework-solutions/skills/tutorials/create-skill/typescript/4-provision-your-azure-resources/ instead call ".\deployment\scripts\publish.ps1"? And if i call the publish script where can I find the values the script asks for?
* Page about adding the skill https://microsoft.github.io/botframework-solutions/skills/tutorials/create-skill/typescript/7-add-your-skill/ says that the localhost mainfest URL is `http://localhost:3978/src/manifest/manifest-1.1.json` which also does not work. n addition on my machine localhost is on port 3980.
EDIT: Just found two other issues mentioning the "manifest.json" problem #3600 and #3601 (with a pull request from August 3rd)

### PR DESCRIPTION
**Pull request by Batta32**
Fix #3638

### Purpose
*What is the context of this pull request? Why is it being done?*
The documentation [TypeScript Update your Skill Manifest](https://microsoft.github.io/botframework-solutions/skills/tutorials/create-skill/typescript/5-update-manifest/), [C# Update your Skill Manifest](https://microsoft.github.io/botframework-solutions/skills/tutorials/create-skill/csharp/5-update-manifest/) and [TypeScript Add your skill to a Virtual Assistant](https://microsoft.github.io/botframework-solutions/skills/tutorials/create-skill/typescript/5-update-manifest/) presents ambiguities described in the issue #3638.

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
- Explain the values of some placeholders
- Add a `publish.ps1` example
- Link to deployment scripts document
- Fix inconsistences of TypeScript manifests

### Tests
*Is this covered by existing tests or new ones? If no, why not?*
\-

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*
\-

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [X] I have updated related documentation

### CODE REVIEW

**Review by peterinnesmsft — APPROVED** [review]
One area

File: docs/_docs/skills/tutorials/create-skill/csharp/5-update-manifest.md [hunk 1]
```diff
@@ -17,4 +17,9 @@ A default manifest describing your Skill is provided as part of the project, you
...
 1. Update `{YOUR_SKILL_APPID}` with the Active Directory AppID of your deployed Skill, you can find this within your `appSettings.json` file.
 
-1. Publish the changes to your Skill endpoint and validate that you can retrieve the manifest using the browser (`/manifest/manifest-1.1.json`)
\ No newline at end of file
+1. Publish the changes to your Skill endpoint and validate that you can retrieve the manifest using the browser (`/manifest/manifest-1.1.json`). Check the [deployment scripts](https://microsoft.github.io/botframework-solutions/virtual-assistant/handbook/deployment-scripts/) document.
+```powershell 
+.\Deployment\Scripts\publish.ps1 -botWebAppName {YOUR_SKILL_BOTWEBAPP_NAME} -resourceGroup {YOUR_RESOURCEGROUP_NAME}
+```
+
+> **Note**: `{YOUR_SKILL_URL}` is the endpoint URL where the Skill will receive the messages (e.g. `http://localhost:3979/api/messages`). Also, `{YOUR_SKILL_APPID}` is the `microsoftAppIp` value, the `{YOUR_SKILL_BOTWEBAPP_NAME}` is the `botWebAppName` and the `{YOUR_RESOURCEGROUP_NAME}` is the `resourceGroupName` that you can find in the `appsettings.json` file populated after the deployment of the Skill.
```
Comment (lauren-mills) [inline docs/_docs/skills/tutorials/create-skill/csharp/5-update-manifest.md]: Small typo here. Should be 'microsoftAppId'

**Review by lauren-mills — APPROVED** [review]

File: docs/_docs/skills/tutorials/create-skill/csharp/5-update-manifest.md (hunk 1 above)
Comment (Batta32) [reply docs/_docs/skills/tutorials/create-skill/csharp/5-update-manifest.md]: Thanks @lauren-mills, we already fixed the typo!

Comment (bhdzllr) [reply docs/_docs/skills/tutorials/create-skill/csharp/5-update-manifest.md]: Nice. I think the Note makes it a lot easier to understand.

Comment (bhdzllr) [reply docs/_docs/skills/tutorials/create-skill/csharp/5-update-manifest.md]: If you deploy to Azure the Skill URL is also the name of the resource from Type "Web App-Bot", should we add this to the note?

**Review by bhdzllr — COMMENTED** [review]
There are two different ports for locahost in this PR (3978, 3979) and in my example project it actually is 3980? What is the default port?

**Comment by Batta32** [comment]
Thanks @bhdzllr for the feedback! We already updated the docs with the information you have provided.

**Review by peterinnesmsft — APPROVED** [review]

**Pull request merged**